File: app/src/main/java/com/pluscubed/velociraptor/api/osm/OsmService.java
package com.pluscubed.velociraptor.api.osm;

import com.pluscubed.velociraptor.api.osm.data.OsmResponse;

import retrofit2.http.Body;
import retrofit2.http.POST;
import rx.Single;

public interface OsmService {
    @POST("interpreter")
    Single<OsmResponse> getOsm(@Body String data);
}

File: app/src/main/java/com/pluscubed/velociraptor/api/osm/OsmService.kt
package com.pluscubed.velociraptor.api.osm

import com.pluscubed.velociraptor.api.osm.data.OsmResponse
import retrofit2.Call
import retrofit2.http.Body
import retrofit2.http.POST

interface OsmService {
    @POST("interpreter")
    fun getOsm(@Body data: String): Call<OsmResponse?>
}
